src/core/components/page-wrapper/TopNavRight.tsx

```diff
@@ -102,12 +108,12 @@ export const TopNavRight = () => {
       </Link>
       <button onClick={handleCommunityClick} className={TOP_NAV_STANDARD_CLASSES}>
         Community
-        <IconVerticalChevron />
+        <IconVerticalChevron isPointingUp={communityChevron} />
```

Comment:
can we use MenuContext instead of local state? maybe compare `openMenuName` to the tab name?
that would also make it more generic instead of needing a state flag per button i think

Reply:
Turns out this was much simpler than I tried to implement

Reply:
great work!